frame/nomination-pools/src/lib.rs

```diff
@@ -411,13 +409,18 @@ pub struct PoolMember<T: Config> {
 	/// Assuming no massive burning events, we expect this value to always be below total issuance.
 	/// This value lines up with the [`RewardPool::total_earnings`] after a member claims a
 	/// payout.
-	pub reward_pool_total_earnings: BalanceOf<T>,
+	pub last_recorded_reward_counter: RewardCounter,
```

Comment:
Should docs here be updated?

Reply:
ah indeed, coming soon in my next commit.